client/stylesheets/_modal.scss

```diff
@@ -823,7 +828,8 @@
 }
 
 /* Hide Colors */
-.sp-palette-row-0 span:nth-child(2), .sp-palette-row-0 span:nth-child(4), .sp-palette-row-0 span:nth-child(5), .sp-palette-row-1 > span:first-child, .sp-palette-row-1 span:nth-child(7) {
+.sp-palette-row-0 span:nth-child(2), .sp-palette-row-0 span:nth-child(4), .sp-palette-row-0 span:nth-child(5), 
```

Comment:
Each selector in a comma sequence should be on its own single line<br>Line contains trailing whitespace